node/src/network.rs

```diff
@@ -99,14 +95,14 @@ pub struct Kadcast<const N: usize> {
     filters: Arc<RwLock<FilterList<N>>>,
     conf: Config,
 
-    counter: AtomicU64,
-
     /// Represents a parsed conf.public_addr
     public_addr: SocketAddr,
+
+    counter: AtomicU64,
```

Comment:
I think this would be clearer as `nonce`, since we're not actually counting anything.

Reply:
I agree
I'll rename it in the next PR to avoid review dismissal